db/index.js

```diff
@@ -0,0 +1,16 @@
+const pg = require('pg');
+
+const pool = new pg.Pool({
+  // Connection info for pool. Note to not call pool.end() till program end.
+  user: 'postgres',
+  host: '127.0.0.1',
```

Comment:
How come this is 127.0.0.1 and not local host?

Reply:
I used 127 because I felt it was more explicit than localhost. I ran into issues with the MySql library using the localhost alias, so I just default to being explicit. Probably not good if someone else configures their localhost to another address, but... like... who DOES that?